### PR TITLE
Fix Comment at target ResolveComReferences

### PR DESCRIPTION
**Pull request by Nirmal4G**
It's actually `InteropOutputPath`

### CODE REVIEW

**Comment by Nirmal4G** [comment]
We really need `[skip ci]`, `[no ci]` or something similar for these kinds of PR!

Ref: microsoft/azure-pipelines-agent#2944

**Comment by Nirmal4G** [comment]
@rainersigwald Split from #5238

File: src/Tasks/Microsoft.Common.CurrentVersion.targets [hunk 1]
```diff
@@ -2666,7 +2666,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
         [IN]
         @(COMReference) - The list of COM references
-        $(BaseIntermediateOutputPath) - The output directory in which to generate wrapper assemblies
+        $(InteropOutputPath) - The output directory in which to generate wrapper assemblies
```
Comment (rainersigwald) [inline src/Tasks/Microsoft.Common.CurrentVersion.targets:2669]:
```suggestion
        $(InteropOutputPath) - The output directory in which to generate wrapper assemblies
        $(IntermediateOutputPath) - Default InteropOutputPath
```

?

Comment (Nirmal4G) [reply src/Tasks/Microsoft.Common.CurrentVersion.targets:2669]:
```suggestion
        $(InteropOutputPath) - The output directory in which to generate wrapper assemblies
                               When $(InteropOutputPath) is not set, then it defaults to $(IntermediateOutputPath).
```

How is this?

Comment (rainersigwald) [reply src/Tasks/Microsoft.Common.CurrentVersion.targets:2669]: Yeah, that looks good.

**Pull request merged**